packages/hebao_v1/contracts/base/BaseWallet.sol

```diff
@@ -209,13 +205,8 @@ abstract contract BaseWallet is ReentrancyGuard, Wallet
         onlyFromFactoryOrModule
         returns (bytes memory returnData)
     {
-        require(
-            !controller.moduleRegistry().isModuleRegistered(to),
-            "TRANSACT_ON_MODULE_DISALLOWED"
-        );
-
         bool success;
-        (success, returnData) = nonReentrantCall(mode, to, value, data);
+        (success, returnData) = _call(mode, to, value, data);
```

Comment:
I don't know how reentrancy can be a problem here. No state is modified (or even read except authorization of the module) in this function so I don't know how it could be abused.

Reply:
It seems you are right, there will be reentrancy, but they will not cause any issues.